adscompstat/tasks.py

```diff
@@ -31,6 +32,18 @@
     Queue("compute-stats", app.exchange, routing_key="compute-stats"),
 )
 
+related_bibstems = []
+related_bibs_file = app.conf.get("JOURNALSDB_RELATED_BIBSTEMS", None)
+if related_bibs_file:
+    try:
+        with open(related_bibs_file, "r") as fj:
+            data = json.load(fj)
+            related_bibstems = data.get("related_bibstems", [])
+    except Exception as err:
+        logger.warning("Unable to load related bibstems list: %s" % err)
+else:
+    logger.warning("Related bibstems filename not set.")
+
 
 # No delay/queue, synchronous only
```

Comment:
Generally, the tasks in `tasks.py` are all async celery tasks - the commenting is clear and makes it obvious that these aren't running via rabbitmq, but I'm curious why these synchronous functions are here instead of at the top of `run.py` or something. Is there a possibility these get moved to an async process at some point?

Reply:
Two reasons: one, that I didn't want to instantiate a sql session manager in both run.py and tasks.py; and two, I was playing around with trying to load the data asynchronously, especially the lists of bibcodes.  I never got it to work correctly here, but eventually that data store going to be in a separate identifiers database rather than in compstat.  When we have that in place, I'll remove these specific tasks.